src/renderer/components/misc/Error.vue

```diff
@@ -35,12 +52,31 @@ window.addEventListener("error", (event) => {
 
     error.value = event;
     console.error(event);
+    log.error(event.message);
     isVisible.value = true;
 });
 
 function onReload() {
     window.document.location.reload();
 }
+
+async function uploadLogsCommand(event) {
+    // Store off event and target to avoid async issue
+    const curE = event;
+    const curTarget = curE.currentTarget;
+
+    // Do the operations
+    try {
+        const url = await uploadLogs();
+        await navigator.clipboard.writeText(url);
+        tooltipMessage.value = "Log URL was copied to clipboard.";
+    } catch (_) {
```

Comment:
nit: you could just `} catch {`, it will also remove warning. Same in [src/renderer/components/navbar/Settings.vue](https://github.com/beyond-all-reason/bar-lobby/pull/330/files#diff-b87832973649a2b045194ffa49bdc5f94f7f2dd5be9fff8f57121850348ebed4)